packages/integration/package.json

```diff
@@ -16,9 +16,9 @@
   "dependencies": {
     "@vanilla-extract/css": "^1.7.2",
     "esbuild": "^0.11.16",
-    "eval": "0.1.6",
+    "eval": "0.1.8",
     "find-up": "^5.0.0",
-    "javascript-stringify": "^2.0.1",
+    "javascript-stringify": "^2.1.0",
```

Comment:
Any reason for this bump? The old range still satisfies the latest version. Feel like this is unnecessary.

Reply:
I thought the [latest release](https://github.com/blakeembrey/javascript-stringify/releases/tag/v2.1.0) was worth upgrading to. The old range matches the latest version, but it still needs a `pnpm update` to update the lockfile.